packages/website/src/examples/layout/ModalWizard/ModalWizardForm.demo.tsx

```diff
@@ -0,0 +1,119 @@
+import {Button, Flex, ModalWizard, ModalWizardStepProps, TextInput, Title} from '@coveord/plasma-mantine';
+import {zodResolver} from '@hookform/resolvers/zod';
+import {useState} from 'react';
+import {Controller, useForm} from 'react-hook-form';
+import {z} from 'zod';
```

Comment:
I understand that you're using `zod` and `react-hook-form` here to mimic the code in the admin but I'm wondering if we should keep our example as Mantine "vanilla" as possible 💭 

It's not blocking for this PR but I'll discuss that with the admin-ui core team

Reply:
That makes sense, I will try to simplify the validation and remove `zod` then.